port_ocean/core/integrations/mixins/sync_raw.py

```diff
@@ -233,13 +250,43 @@ async def register_raw(
             resource for resource in config.resources if resource.kind == kind
         ]
 
-        return await asyncio.gather(
+        diffs: list[EntitySelectorDiff] = await asyncio.gather(
             *(
                 self._register_resource_raw(resource, results, user_agent_type, True)
                 for resource in resource_mappings
             )
         )
 
```

Comment:
will this be optimized in your other PR? as there could be thousand of entities at once and spawning all can have a huge decrease in performance and take a lot of memory

Reply:
Did he change anything?

Reply:
i didn't changed anything there, if it causes weak performance i'll handle it in the other PR but there isn't suppose to be any new performance issue from the current PR